pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -597,30 +597,44 @@ const (
 // Name - The PV name.
 // Capacity - The PV storage capacity.
 // StorageClass - The PV storage class name.
-// Action - The PV migration action (move|copy)
-// SupportedActions - The list of supported actions.
+// Supported - Lists of what is supported
+// Selection - Choices made from supported
 // staged - A PV has been explicitly added/updated.
 type PV struct {
-	Name             string            `json:"name,omitempty"`
-	Capacity         resource.Quantity `json:"capacity,omitempty"`
-	StorageClass     string            `json:"storageClass,omitempty"`
-	SupportedActions []string          `json:"supportedActions"`
-	Action           string            `json:"action,omitempty"`
-	PVC              PVC               `json:"pvc,omitempty"`
-	staged           bool
+	Name         string            `json:"name,omitempty"`
+	Capacity     resource.Quantity `json:"capacity,omitempty"`
+	StorageClass string            `json:"storageClass,omitempty"`
+	Supported    Supported         `json:"supported"`
```

Comment:
You can probably just include the `Supported` and `Selection` inline since you are not renaming them.